### ISSUE TITLE
[Proposal] Mobile App for Retail POS

### ISSUE THREAD

**Issue by rmehta** [frappe/erpnext#15946]
App for retail point-of-sale (billing)

1. Start a new bill
2. Add items (select, scan barcode/qrcode?)
3. Add discounts
5. Add custom notes
4. Add payment details
5. Sync and push invoice

**Comment by palashjhabak** [comment]
I was just wondering what would be the best mobile based UI to achieve this ? I am looking at other mobile based POS for reference, but if there is already some guideline/thoughts on this, please let us know.

**Comment by seehive** [comment]
i think PhoneGap or ionic would be a better choice and i will consider working on these mobile apps and any workflow idea will be helpful

**Comment by palashjhabak** [comment]
@seehive Why not having it native using React Native and Frappe REST APIs ?

**Comment by seehive** [comment]
@palashjhabak yes, we can also consider React Native and definitely we will be using Frappe REST APIs,, i am trying to get some time to prepare some design ideas so that we can have some starting point

**Comment by seehive** [comment]
any specific reason about Kotlin Native? i think React Native will server the same purpose but i would love to know the Frappe team point of view

**Comment by magic-overflow** [comment]
Frappe Team meantioned in the discussion.
https://discuss.erpnext.com/t/sales-person-tracking/42605/4

**Comment by palashjhabak** [comment]
@magic-overflow ohh ok. Any major advantages of Kotlin Native over React Native ?

**Comment by magic-overflow** [comment]
I have no experience in these both languages. You can join discussion with frappe team.

**Comment by howdini** [comment]
There is an ERPNext Android Mobile POS app on playstore. The current version is absolutely free...  https://play.google.com/store/apps/details?id=com.bslkenya.sukaripos
